app/angular/src/client/preview/angular-beta/StorybookWrapperComponent.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  AfterViewInit,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  Inject,
+  OnDestroy,
+  Type,
+  ViewChild,
+  ViewContainerRef,
+} from '@angular/core';
+import { Subscription, Subject } from 'rxjs';
+import { map, skip } from 'rxjs/operators';
+
+import { ICollection } from '../types';
+import { STORY_PROPS } from './StorybookProvider';
+import { ComponentInputsOutputs, getComponentInputsOutputs } from './utils/NgComponentAnalyzer';
+import { RendererService } from './RendererService';
+
+const getNonInputsOutputsProps = (
+  ngComponentInputsOutputs: ComponentInputsOutputs,
+  props: ICollection = {}
+) => {
+  const inputs = ngComponentInputsOutputs.inputs
+    .filter((i) => i.templateName in props)
+    .map((i) => i.templateName);
+  const outputs = ngComponentInputsOutputs.outputs
+    .filter((o) => o.templateName in props)
+    .map((o) => o.templateName);
+  return Object.keys(props).filter((k) => ![...inputs, ...outputs].includes(k));
+};
+
+/**
+ * Wraps the story template into a component
+ *
+ * @param storyComponent
+ * @param initialProps
+ */
+export const createStorybookWrapperComponent = (
+  template: string,
+  storyComponent: Type<unknown>,
+  styles: string[],
+  initialProps?: ICollection
+): Type<any> => {
+  @Component({
+    selector: RendererService.SELECTOR_STORYBOOK_WRAPPER,
+    template,
+    styles,
+  })
+  class StorybookWrapperComponent implements AfterViewInit, OnDestroy {
+    private storyPropsSubscription: Subscription;
+
+    @ViewChild(storyComponent ?? '', { static: true }) storyComponentElementRef: ElementRef;
```

Comment:
Why do you need this? What can I expect from a `@ViewChild('')`?

Reply:
Before there were 2 wrappers (one for component and one for template). 
There is now only 1 for both. And in case of a story without component the ViewChild are "disabled" with `''` 

This change has related to this open PR https://github.com/storybookjs/storybook/pull/13507
(I'm based on this one)